src/modules/remove.js

```diff
@@ -0,0 +1,22 @@
+export default (description) => {
+  const toModify = JSON.parse(localStorage.getItem('tasks'));
+  let ind;
+  toModify.forEach((element) => {
+    if (element.description === description) {
+      ind = element.index;
+    }
+  });
+  toModify.splice(ind - 1, 1);
+  let newIndex = 1;
+  toModify.forEach((element) => {
+    element.index = newIndex;
+    newIndex += 1;
+  });
+
+  const toStorage = JSON.stringify(toModify);
+  localStorage.setItem('tasks', toStorage);
+  const trashCans = document.querySelectorAll('.trash-can');
+  trashCans.forEach((can) => {
+    can.parentElement.remove();
+  });
+};
```

Comment:
- [ ] I am making this a required change because I noticed that you changed the operational logic for most/all the functionalities in this project. Deleting a task based on the description will produce bugs when I try to delete a task that has the same description with one or two other tasks. It also means that I could be deleting the last task with the same description whereas my intention is delete the first task with the same description. 😎 

For example, let's say your `todoArray` is similar to this:
```JavaScript
[
  {
    index: 1,
    description: 'Code for 5 hours',
    completed: false,
  },
  {
    index: 2,
    description: 'Play soccer',
    completed: false,
  },
  {
    index: 3,
    description: 'Take a walk in the park',
    completed: false,
  },
  {
    index: 4,
    description: 'Play soccer',
    completed: false,
  },
];
```
When I try to delete the task with **_Play soccer_** that has an `index` of **_2_**, I will encounter an issue. Based on the logic for deleting a task, the `ind` variable in the function `remove` in the `src/modules/remove.js` file will be equal to `4` instead of `2` because of the duplicate descriptions. Although the task _**Play soccer**_ with the index of `2` will be removed at the moment because of the `removeThrash` function in the `src/modules/removeTrash.js` file, I would have removed the task _**Play soccer**_ with the index of `4` and that is what will be saved in the `localStorage`. Therefore, when I reload the page, it will be evident that I removed the wrong task. 

#### To fix this, I will suggest that you use the task's `index` property to remove tasks since they are unique. Kindly refactor the logic and write a corresponding unit test. 👍 💻